public/app.js

```diff
@@ -0,0 +1,330 @@
+// AUTH CONTROLLER
+var authController = (function() {
+    return {
+        //Authentication state listener
+        initApp: function() {
```

Comment:
Is this really initing the app? Or is it just getting a logged in user?

login() might actually be a better name, or get_user()

Reply:
Its getting a logged in user. I'll go with get_user for clarity